cmd/extensions/oidc-policy/internal/controller/oidcpolicy_reconciler.go

```diff
@@ -227,7 +232,16 @@ func (r *OIDCPolicyReconciler) reconcileMainAuthPolicy(ctx context.Context, pol
 }
 
 func (r *OIDCPolicyReconciler) reconcileCallbackAuthPolicy(ctx context.Context, pol *v1alpha1.OIDCPolicy, igw *ingressGatewayInfo) (*kuadrantv1.AuthPolicy, error) {
-	desiredAuthPol, err := buildCallbackAuthPolicy(pol, igw)
+	tokenExchangeOpts := make(map[string]string)
```

Comment:
nit:

Would you mind please naming this perhaps _token endpoint_ options (or yet _token request_ options), instead of _token exchange_ options?

"Token exchange" drives my mind to a different OAuth concept. If I understand it correctly, here we're talking about exchanging the authorisation _code_ (not _token_) for an access token. As in the token request step of the authorisation code grant, right?